sample/src/main/java/com/facebook/samples/litho/playground/PlaygroundComponentSpec.java

```diff
@@ -21,13 +23,16 @@
 
 import com.facebook.litho.annotations.LayoutSpec;
 import com.facebook.litho.annotations.OnCreateLayout;
-import com.facebook.litho.widget.Text;
 
 @LayoutSpec
 public class PlaygroundComponentSpec {
 
+  @PropDefault(resType = ResType.DIMEN_SIZE)
```

Comment:
Could you revert these changes? Good to see example used but we don't want to add a bunch of stuff to the playground.